api/v1/views/index.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python3
+
+from api.v1.views import app_views
+from flask import jsonify
+
+
+@app_views.route('/status', strict_slashes=False)
+def status():
+    return jsonify({'status': "OK"})
```

Comment:
can you please add description for this code?

Reply:
Of course, I'll do this changes in the next commit.